### ISSUE TITLE
CopyPkgJsonPlugin can't find path to package.json with the new Webpack 5

### ISSUE THREAD

**Issue by RosenTomov** [pmsandhu/copy-pkg-json-webpack-plugin#8]
After updating to the new Webpack 5, CopyPkgJsonPlugin can no longer find the package.json
I've tried manually passing the path (`path.resolve('./')`, `process.cwd()`), but it didn't help.

I'm using it to clear out dev dependencies
```
new CopyPkgJsonPlugin({
  remove: ['devDependencies'],
}),
```

```
"webpack": "5.1.3",
"copy-pkg-json-webpack-plugin": "0.0.39",
```

And here's the error:
```
[webpack-cli] Error:   Cannot find the following package.json path: E:\GitHub\TestProject\package.json
  If your package.json is not in the root directory of the current node process
  pass the path to where it is located as the second argument to the plugin:
    ie. CopyPkgJsonPlugin({options}, 'path/main')
    at apply (E:\GitHub\TestProject\node_modules\copy-pkg-json-webpack-plugin\dist\index.js:1:1361)
    at createCompiler (E:\GitHub\TestProject\node_modules\webpack\lib\webpack.js:71:12)
    at create (E:\GitHub\TestProject\node_modules\webpack\lib\webpack.js:115:15)
    at webpack (E:\GitHub\TestProject\node_modules\webpack\lib\webpack.js:139:31)
    at f (E:\GitHub\TestProject\node_modules\webpack\lib\index.js:35:15)
    at WebpackCLI.createCompiler (E:\GitHub\TestProject\node_modules\webpack-cli\lib\webpack-cli.js:219:24)
    at WebpackCLI.run (E:\GitHub\TestProject\node_modules\webpack-cli\lib\webpack-cli.js:248:31)
    at async runCLI (E:\GitHub\TestProject\node_modules\webpack-cli\lib\bootstrap.js:76:24)
```

The previous version works fine.
```
"webpack": "4.44.2",
```

**Comment by RosenTomov** [comment]
So I found some time and got around to this again
https://github.com/pmsandhu/copy-pkg-json-webpack-plugin/blob/master/src/index.js#L27

`compiler.inputFileSystem._statSync(root)`
should be 
`compiler.inputFileSystem.statSync(root)`

**Comment by rotoclabs** [comment]
> So I found some time and got around to this again
> https://github.com/pmsandhu/copy-pkg-json-webpack-plugin/blob/master/src/index.js#L27
> 
> `compiler.inputFileSystem._statSync(root)`
> should be
> `compiler.inputFileSystem.statSync(root)`

I could use another plugin with a good success.

```
const CopyPlugin = require('copy-webpack-plugin');
module.exports = {
    //
    plugins: [
        new CopyPlugin({
            patterns: [
                {
                    from: 'package.json',
                    to: 'package.json',
                    transform: (content, path) => {
                        jsonContent = JSON.parse(content);
                        delete jsonContent.devDependencies;
                        delete jsonContent.scripts;
                        return JSON.stringify(jsonContent, null, 2);
                    }
                }
            ]
        })
    ]
};
```

**Comment by martinratinaud** [comment]
@RosenTomov do you plan on fixing this '_statSync` thing soon or do you need a PR?

Thanks

**Comment by pmsandhu** [comment]
Apologies. I just saw this issue. Unsure why GitHub does not send me notifications regarding issues for this repo. 

Anyways, I have implemented your fix to make it compatible with  webpack 5 and published a new version `0.0.40` to npm. 

Thank you for finding this.
